middleware_jwt_test.go

```diff
@@ -179,10 +179,10 @@ var jwtWithCentralDefNoPolicyBaseField = `
 
 `
 
-const JWTSECRET = "9879879878787878"
+const jwtSecret = "9879879878787878"
 
 // openssl genrsa -out app.rsa
-const JWTRSA_PRIVKEY = `
+const jwtRSAPrivKey = `
```

Comment:
This doesn't apply to Go at all, because capitalized means exported.

There isn't a consensus about uppercase vs camelcase for constants, but ingeneral people use camelcase too.